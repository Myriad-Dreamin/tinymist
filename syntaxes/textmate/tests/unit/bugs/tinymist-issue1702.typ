#let quote(x) = $\"#x\"$
Rest of code here...
