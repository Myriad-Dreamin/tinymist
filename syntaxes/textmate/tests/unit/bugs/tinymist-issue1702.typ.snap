>#let quote(x) = $\"#x\"$
#^ source.typst keyword.control.hash.typst
# ^^^ source.typst meta.expr.let.typst storage.type.typst
#    ^ source.typst meta.expr.let.typst
#     ^^^^^ source.typst meta.expr.let.typst entity.name.function.typst
#          ^ source.typst meta.expr.let.typst meta.brace.round.typst
#           ^ source.typst meta.expr.let.typst variable.other.readwrite.typst
#            ^ source.typst meta.expr.let.typst meta.brace.round.typst
#             ^ source.typst meta.expr.let.typst
#              ^^ source.typst meta.expr.let.typst keyword.operator.assignment.typst
#                ^ source.typst meta.expr.let.typst markup.math.typst punctuation.definition.string.begin.math.typst
#                 ^^ source.typst meta.expr.let.typst markup.math.typst constant.character.escape.content.typst
#                   ^ source.typst meta.expr.let.typst markup.math.typst variable.other.readwrite.hash.typst
#                    ^ source.typst meta.expr.let.typst markup.math.typst variable.other.readwrite.typst
#                     ^^ source.typst meta.expr.let.typst markup.math.typst constant.character.escape.content.typst
#                       ^ source.typst meta.expr.let.typst markup.math.typst punctuation.definition.string.end.math.typst
>Rest of code here...
#^^^^^^^^^^^^^^^^^ source.typst
#                 ^^^ source.typst punctuation.definition.ellipsis.typst
>